ocaml-lsp-server/src/ocamlformat.mli

```diff
@@ -1,44 +0,0 @@
-open Import
-
-module File_type : sig
-  type t =
-    | Impl
-    | Intf
-    | Name of string
-end
-
-module Input : sig
-  type t =
-    | Stdin of string * File_type.t
-    | File of string
-end
-
-module Output : sig
-  type _ t =
-    | File : string -> unit t
-    | Stdout : string t
-end
-
-module Options : sig
```

Comment:
> We will have to set --lines option to support textDocument/{rangeFormatting, onTypeFormatting}, so I'll leave minimal support of passing options instead of completely removing it.

It should be quite simple to add back. We'll just ned to pass a range in the main function and then change `args` to generate arguments for that range.